lib/validation/validators/maxLength.ts

```diff
@@ -1,3 +1,9 @@
-export function maxLength(value: string, length: number): boolean {
+import { isArray, isString } from '../../support/Utils'
+
+export function maxLength(value: unknown, length: number): boolean {
+  if (!(isString(value) || isArray(value))) {
+    return false
+  }
+
   return value.length <= length
 }
```

Comment:
Should this validator accept any value that has `length` property besides string and array?

Reply:
Wouldn't matter much in my opinion. We aren't using this for anything else than string.

Reply:
@kiaking Any thoughts?

Reply:
Ah, sorry you already replied to another thread 🙌